<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕코딩</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>EMS SCADA</string>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="totalFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1181</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="headframe">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1181</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : rgb(41, 42, 55);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="iconFrame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>8</y>
        <width>71</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/iot);
background-repeat: no-repeat;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblTitle">
      <property name="geometry">
       <rect>
        <x>103</x>
        <y>23</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans Ultra Bold</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color : rgb(255,255,255);</string>
      </property>
      <property name="text">
       <string>EMS IoT DASHBOARD</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="greetingFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>271</width>
       <height>491</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(60, 60, 90);
border-style: outset;
border-width: 5px;
border-color:rgb(0,0,255);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="weatherFrame">
      <property name="geometry">
       <rect>
        <x>103</x>
        <y>197</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/sun);
backgroud-repeat: no-repeat;
border:none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblDate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,170,0);
border : none;</string>
      </property>
      <property name="text">
       <string>9999-99-99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblDay">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>80</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,170,0);
border : none;</string>
      </property>
      <property name="text">
       <string>Wendnesday</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,170,0);
border : none;</string>
      </property>
      <property name="text">
       <string>99:99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblGreeting">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(210, 210, 210);
border : none;</string>
      </property>
      <property name="text">
       <string>Good Afternoon</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>lblDay</zorder>
     <zorder>weatherFrame</zorder>
     <zorder>lblDate</zorder>
     <zorder>lblTime</zorder>
     <zorder>lblGreeting</zorder>
    </widget>
    <widget class="QFrame" name="tempFrame">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>321</width>
       <height>491</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(72, 0, 0);
border-style: outset;
border-width: 5px;
border-color:rgb(180, 50, 0);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblCurrTemp">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>13</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
border : none;</string>
      </property>
      <property name="text">
       <string>EMS99 Temperature</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTempTitle">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>165</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT</family>
        <pointsize>60</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 100, 20);
border : none;</string>
      </property>
      <property name="text">
       <string>99.9 </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTempTitle_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>51</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 100, 20);
border : none;</string>
      </property>
      <property name="text">
       <string>℃</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>301</width>
        <height>171</height>
       </rect>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="humidIconFrame_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/temp);
background-repeat: no-repeat;
border:none;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <zorder>lblTempTitle</zorder>
     <zorder>lblTempTitle_2</zorder>
     <zorder>dial</zorder>
     <zorder>humidIconFrame_2</zorder>
     <zorder>lblCurrTemp</zorder>
    </widget>
    <widget class="QFrame" name="tempFrame_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>90</y>
       <width>321</width>
       <height>491</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 50, 115);
border-style: outset;
border-width: 5px;
border-color:rgb(0, 50, 180);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblCurrTemp_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>13</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
border : none;</string>
      </property>
      <property name="text">
       <string>EMS99 Humidity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTempTitle_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>51</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(221, 235, 253);
border : none;</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="humidIconFrame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>171</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/humid);
background-repeat: no-repeat;
border:none;</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QDial" name="dial_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>301</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">widget-color: rgb(221, 235, 253);</string>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblTempTitle_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT</family>
        <pointsize>60</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(221, 235, 253);
border : none;</string>
      </property>
      <property name="text">
       <string>99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <zorder>humidIconFrame</zorder>
     <zorder>lblCurrTemp_3</zorder>
     <zorder>lblTempTitle_6</zorder>
     <zorder>dial_3</zorder>
     <zorder>lblTempTitle_5</zorder>
    </widget>
    <zorder>headframe</zorder>
    <zorder>greetingFrame</zorder>
    <zorder>tempFrame_2</zorder>
    <zorder>tempFrame</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="dashboard.qrc"/>
 </resources>
 <connections/>
</ui>
